Framework/Algorithms/src/GetDetectorOffsets.cpp

```diff
@@ -407,11 +470,11 @@ double GetDetectorOffsets::fitSpectra(
 //-----------------------------------------------------------------------------------------
 /** Create a function string from the given parameters and the algorithm inputs
  * @brief GetDetectorOffsets::createFunction
- * @param peakHeight
- * @param peakLoc
- * @param peakSigma
- * @param a0
- * @param a1
+ * @param peakHeight : peak height
```

Comment:
The names of the parameters are clear enough that that don't need to be documented.